ScienceJournal/UI/TrialDetailViewController.swift

```diff
@@ -1296,17 +1293,25 @@ class TrialDetailViewController: MaterialHeaderViewController,
 
     // Edit start time.
     actions.append(PopUpMenuAction(title: String.editCropStartTime, isEnabled: true) { _ in
-      self.cropRangeController.showTimestampEditAlert(forCropMarkerType: .start)
+      self.showTimestampEditAlert(for: .start)
     })
 
     // Edit end time.
     actions.append(PopUpMenuAction(title: String.editCropEndTime, isEnabled: true) { _ in
-      self.cropRangeController.showTimestampEditAlert(forCropMarkerType: .end)
+      self.showTimestampEditAlert(for: .end)
     })
 
     return actions
   }
 
+  private func showTimestampEditAlert(for cropMarkerType: CropOverlayView.MarkerType) {
+    guard let trialCropRange = self.currentPlaybackViewController?.1.cropRange else {
+      return
```

Comment:
I don't think this should ever be possible. I considered doing a `fatalError` here, but this seemed safer, given the uncertainty.